.swiftlint.yml

```diff
@@ -1,11 +1,3 @@
-disabled_rules:
```

Comment:
So we don't want you guys to alter these rules without us giving to go ahead 😄 please undo these changes.

Reply:
Done👍🏻